pkg/apis/storage/types.go

```diff
@@ -390,6 +390,20 @@ type CSIDriverSpec struct {
 	//
 	// +optional
 	RequiresRepublish *bool
+
+	// SELinuxMountSupported specifies if the CSI driver supports "-o context"
+	// mount option.
+	//
+	// When "true", Kubernetes may call NodeStage / NodePublish with "-o context=xyz" mount
+	// option. The CSI driver must ensure that all volumes can be mounted with different
+	// `-o context` options. This is typical for storage backends that provide volumes
+	// as filesystems on block devices or as independent shared volumes.
+	//
+	// When "false", Kubernetes won't pass any special SELinux mount options to the driver.
+	// This is typical for volumes that represent subdirectories of a bigger shared filesystem.
+	//
+	// Default is "false".
+	SELinuxMountSupported *bool
```

Comment:
To align with field naming with existing fields, I suggest removing  "Supported" from the field name.

Reply:
Naming is hard. Would plain `SELinuxMount` imply that all volumes are mounted with SELinux opts? The field only allows kubelet to use `mount -o context`, it does not mandate it - a pod + volume must meet quite lot of conditions before kubelet can use it.

Reply:
Renamed to `SELinuxMount`